integration/loader-test.ts

```diff
@@ -103,5 +124,12 @@ test.describe("loader in an app", () => {
     let app = new PlaywrightFixture(appFixture, page);
     await app.goto("/redirect");
     expect(await app.getHtml()).toMatch(HOME_PAGE_TEXT);
+    expect(await app.getHtml()).toMatch(REDIRECT_TARGET_TEXT);
```

Comment:
Strengthened the existing redirect tests a bit to ensure it renders the right content at the redirected location.